yt/data_objects/derived_quantities.py

```diff
@@ -135,7 +135,7 @@ class WeightedAverageQuantity(DerivedQuantity):
     >>> ad = ds.all_data()
     >>> print ad.quantities.weighted_average_quantity([("gas", "density"),
     ...                                                ("gas", "temperature")],
-    ...                                               ("gas", "cell_mass"))
+    ...                                               ("gas", "mass"))
```

Comment:
```suggestion
    ...                                                ("gas", "mass"))
```

Reply:
I resolved this one when I fixed confilcts with `main`.